zigpy/zcl/__init__.py

```diff
@@ -972,17 +989,22 @@ def general_command(
 
     def send_default_rsp(
         self,
-        hdr: foundation.ZCLHeader,
+        tsn: t.uint8_t,
+        command_id: t.uint8_t,
```

Comment:
Unwrapping  `hdr` here, as `tsn` and `command_id` are used from it. A caller in `security.py` wrongly assumed that changing direction in `hdr` will have an effect on this function